interactive_engine/src/lgraph/include/db/common/types.h

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright 2021 Alibaba Group Holding Limited.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <cstddef>
+#include <cstdint>
+#include <limits>
+
+#include "db/common/namespace.h"
+
+namespace DB_NAMESPACE {
+
+typedef uint64_t SnapshotId;
+typedef uint32_t LabelId;
+typedef uint64_t VertexId;
+typedef uint64_t EdgeInnerId;
+typedef uint32_t PropertyId;
+typedef size_t SerialId;
```

Comment:
Using uint32_t instead of  size_t for SerialId?

Reply:
consistent with the encoding in rust end, or replace them at both side